modules/dynamic-provisioning-change-default-class.adoc

```diff
@@ -29,7 +27,7 @@ standard             kubernetes.io/aws-ebs
 ----
 <1> `(default)` denotes the default storage class.
 
-. Change the value of the annotation
+. Change the value of the annotation:
```

Comment:
I would remove this colon. It's unnecessary, and it also puts two colons in one sentence.

Reply:
I have used colons in procedural steps , is there a specific style guideline that mentions about colon usage?  I  checked the IBM style guide.

Reply:
@stevsmit

Reply:
This sentence uses two colons: 

Change the value of the annotation: storageclass.kubernetes.io/is-default-class to false for the default storage class:

It's not necessary to have one after `annotation`. It just interrupts readability and doesn't really serve a purpose. :D

Change the value of the annotation `storageclass.kubernetes.io/is-default-class` to `false` for the default storage class:

Alternatively, you could say: 

Change the value of the `storageclass.kubernetes.io/is-default-class` annotation to `false` for the default storage class:

@Amrita42 lmk what you think

Reply:
I prefer >> Change the value of the storageclass.kubernetes.io/is-default-class annotation to false for the default storage class:  
to keep it consistent with your suggestion for line 39.